pyproject.toml

```diff
@@ -76,11 +77,11 @@ power_grid_model_io = ["*.yaml"]
 
 [tool.pytest.ini_options]
 testpaths = ["tests/unit"]
-addopts = ["--cov=src/power_grid_model_io", "--cov-report", "term", "--cov-fail-under=20"]
+addopts = ["--cov=src/power_grid_model_io", "--cov-report", "term", "--cov-report", "html:python_cov_html", "--cov-report", "xml:python_coverage.xml", "--cov-fail-under=20"]
```

Comment:
Do we really need the html covarage report?
Can we name the coverage report `coverage.xml`; this is a python only project.

Reply:
xml is needed later for sonarcloud.

html is not per se needed. I personally find it nice to read the coverage report in html.

Reply:
removed python prefix.